docs/sphinx/quickstart.rst

```diff
@@ -53,59 +54,107 @@ versions, is available at :ref:`prerequisites`.
 Building |hpx|
 ==============
 
-Once you have the source code and the dependencies, set up a separate build
-directory and configure the project. Assuming all your dependencies are in paths
+Once you have the source code and the dependencies and assuming all your dependencies are in paths
 known to |cmake|, the following gets you started:
 
-.. code-block:: sh
+#. First, set up a separate build directory to configure the project:
 
-    # In the HPX source directory
-    mkdir build && cd build
-    cmake -DCMAKE_INSTALL_PREFIX=/install/path ..
-    make install
+  .. code-block:: sh
 
-This will build the core |hpx| libraries and examples, and install them to your
-chosen location. If you want to install |hpx| to system folders, simply leave out
-the ``CMAKE_INSTALL_PREFIX`` option. This may take a while. To speed up the
-process, launch more jobs by passing the ``-jN`` option to ``make``.
+      cd ..
+      mkdir build && cd build
 
-.. tip::
+#. To configure the project you have the following options:
 
-   Do not set only ``-j`` (i.e. ``-j`` without an explicit number of jobs)
-   unless you have a lot of memory available on your machine.
+  * To build the core |hpx| libraries and examples, and install them to your
+  chosen location (suggested):
```

Comment:
```suggestion
  chosen location (recommended):
```
? I may just be missing what `suggested` is referring to.

Reply:
I wanted to mention that this step is the recommended one. Do you think I should remove it because it's not clear?

Reply:
Nope, you can leave it. I would just change the word to `recommended` rather than `suggested`.